remoteappmanager/templates/error.html

```diff
@@ -1,28 +1,23 @@
 {% extends "page.html" %}
 
 {% block login_widget %}
-{% endblock %}
+{% end %}
 
 {% block main %}
 
 <div class="error">
   {% block h1_error %}
-  <h1>
-    {{status_code}} : {{status_message}}
-  </h1>
-  {% endblock h1_error %}
+    <h1>
+      {{status_code}} : {{status_message}}
+    </h1>
+  {% end %}
   {% block error_detail %}
-  {% if message %}
-  <p>
-    {{message}}
-  </p>
-  {% endif %}
-  {% if message_html %}
```

Comment:
I couldn't find any other reference to `message_html`, maybe an old unused anymore variable ?